code/game/objects/structures/crates_lockers/closets/secure/personal.dm

```diff
@@ -12,6 +12,7 @@
 		new /obj/item/weapon/storage/backpack(src)
 	else
 		new /obj/item/weapon/storage/backpack/satchel(src)
+	new /obj/item/device/instrument/piano_synth(src)

```

Comment:
Give it like a 25% prob of appearing imo. That way you actually have to search them out.

Unless you want the station filled with these things, since most stations have between 4 and 10 personal closets.